pipelines/ocw/pipeline.yml

```diff
@@ -0,0 +1,76 @@
+---
+resource_types:
+  - name: rclone
+    type: docker-image
+    source:
+      repository: sothr/concourse-rclone-resource
+      tag: latest
+resources:
+  - name: ol-infrastructure
+    type: git
+    source:
+      uri: https://github.com/mitodl/ol-infrastructure.git
+      branch: ((ol-infrastructure-git-ref))
+  - name: ocw-www
+    type: git
+    source:
+      uri: https://github.com/mitodl/ocw-www.git
+      branch: ((ocw-www-git-ref))
+  - name: ocw-hugo-themes
+    type: git
+    source:
+      uri: https://github.com/mitodl/ocw-hugo-themes.git
+      branch: ((ocw-hugo-theme-git-ref))
+  - name: ocw-site
+    type: rclone
+    source:
+      config: |
+        [s3-remote]
+        type = s3
+        provider = AWS
+        env_auth = true
+        region = us-east-1
+
+jobs:
+  - name: build
+    serial: true
+    plan:
+      - get: ol-infrastructure
+      - get: ocw-www
+      - get: ocw-hugo-themes
```

Comment:
It's probably worth having this and the ocw-www repositories trigger the pipeline build. As this is defined currently it will only build if it gets executed manually, unless we set up webhooks from GitHub to trigger these resources.

Reply:
:+1: I was planning to add `trigger: true` at some point, when I could be confident about it running automatically.

Reply:
I've added `trigger: true` in 71d76bdff3dbe3dbe4e28533bd67b7f88c33494f